lib/sarif/base_sarif.rb

```diff
@@ -166,6 +166,15 @@ def sarif_level(severity)
       end
     end
 
+    def self.passed?(sarif)
```

Comment:
Can we change Sarif::BaseSarif.passed? to Sarif::BaseSarif.salus_passed?

That might better convey the nuance between no SARIF issues found vs issues only found from enforced scanners [our salus logic for pass / fail]